src/_components/course-item.component.html

```diff
@@ -0,0 +1,31 @@
+<div class="<!-- @@var classItemCourses -->">
+    <div class="<!-- @@var classThumb -->">
```

Comment:
We don't need to use `<!-- @@var classThumb -->` for this one.
Please use the class=`title-info` here,
Same below.
Because the component need to be enough style, color...
If you want to custom it, you can use classNames of parent node